File: core/src/main/java/in/koreatech/koin/core/networks/responses/DefaultResponse.java
package in.koreatech.koin.core.networks.responses;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by hyerim on 2018. 5. 28....
 */
public class DefaultResponse {
    @SerializedName("success")
    @Expose
    public boolean success;

    @SerializedName("message")
    @Expose
    public String message;

    @SerializedName("error")
    @Expose
    public String error;

    @SerializedName("code")
    @Expose
    public String code;

    @SerializedName("grantEdit")
    @Expose
    public boolean isGrantEdit;


    public boolean getSuccess() {
        return success;
    }

    public String getMessage() {
        return message;
    }

    public String getError() {
        return error;
    }

    public String getCode() {
        return code;
    }

    public boolean isGrantEdit() {
        return isGrantEdit;
    }
}

File: data/src/main/java/in/koreatech/koin/data/response/user/DefaultResponse.kt
package `in`.koreatech.koin.data.response.user

import com.google.gson.annotations.SerializedName

data class DefaultResponse(
    @SerializedName("success") val success: Boolean,
    @SerializedName("message") val message: String,
    @SerializedName("error") val error: String,
    @SerializedName("code") val code: String,
    @SerializedName("grantEdit") val isGrantEdit: Boolean
)
